shell/src/main/java/tachyon/shell/command/FreeCommand.java

```diff
@@ -61,4 +61,9 @@ void runCommand(TachyonURI path) throws IOException {
   public String getUsage() {
     return "free <file path|folder path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Frees the given file or directory.";
```

Comment:
Be more descriptive like "Removes the file or directory(recursively) from Tachyon memory space."